rocAL/rocAL_pybind/rocal_pybind.cpp

```diff
@@ -422,13 +639,93 @@ namespace rocal{
-        m.def("rocalCopyToOutputTensor",&wrapper_tensor);
-        m.def("rocalCopyToOutputTensor32",&wrapper_tensor32);
-        m.def("rocalCopyToOutputTensor16",&wrapper_tensor16);
-        m.def("rocalCopyCupyToOutputTensor32",&wrapper_copy_cupy_tensor32);
-        m.def("rocalCopyCupyToOutputTensor16",&wrapper_copy_cupy_tensor16);
+        m.def("rocalGetOutputTensors", [](RocalContext context) {
+            rocalTensorList * tl = rocalGetOutputTensors(context);
+            py::list list;
+            unsigned int size_of_tensor_list = tl->size();
+            for (uint i =0; i< size_of_tensor_list; i++)
+                list.append(tl->at(i));
+            return list; 
+        });
+        m.def("rocalGetImageLabels", [](RocalContext context) {
+            rocalTensorList *labels = rocalGetImageLabels(context);
+            return py::array(py::buffer_info(
+                            (int *)(labels->at(0)->buffer()),
+                            sizeof(int),
+                            py::format_descriptor<int>::format(),
+                            1,
+                            {labels->size()},
+                            {sizeof(int) }));
+        });
+        m.def("rocalGetBoundingBoxLabels", [](RocalContext context) {
+            rocalTensorList *labels = rocalGetBoundingBoxLabel(context);
+            py::list labels_list;
+            py::array_t<int> labels_array;
+            for (int i = 0; i < labels->size(); i++) {
+                int *labels_buffer = (int *)(labels->at(i)->buffer());
+                labels_array = py::array(py::buffer_info(
+                            (int *)(labels->at(i)->buffer()),
+                            sizeof(int),
+                            py::format_descriptor<int>::format(),
+                            1,
+                            {labels->at(i)->info().dims().at(0)},
+                            {sizeof(int) }));
+                labels_list.append(labels_array);
+            }
+            return labels_list;
+        });
+        m.def("rocalGetBoundingBoxCords", [](RocalContext context) {
+            rocalTensorList *boxes = rocalGetBoundingBoxCords(context);
+            py::list boxes_list;
+            py::array_t<double> boxes_array;
+            for (int i = 0; i < boxes->size(); i++) {
+                double *box_buffer = (double *)(boxes->at(i)->buffer());
+                boxes_array = py::array(py::buffer_info(
+                            (double *)(boxes->at(i)->buffer()),
+                            sizeof(double),
+                            py::format_descriptor<double>::format(),
+                            1,
+                            { boxes->at(i)->info().dims().at(0) * 4},
+                            {sizeof(double) }));
+                boxes_list.append(boxes_array);
+            }
+            return boxes_list;
+        });
+        m.def("rocalGetMatchedIndices", [](RocalContext context) {
```

Comment:
Please remove the wrappers. If you think it will introduce more file changes add a block comment which will show only 2 lines of change.

Reply:
Done